pvlib/pvsystem.py

```diff
@@ -1373,6 +1410,121 @@ def get_iam(self, aoi, iam_model='physical'):
             raise ValueError(model + ' is not a valid IAM model')
 
 
+@dataclass
+class AbstractMount(ABC):
+
+    @abstractmethod
+    def calculate_orientation(self, solar_zenith, solar_azimuth):
+        """
+        Determine module orientation.
+
+        Parameters
+        ----------
+        solar_zenith : numeric
+            Solar apparent zenith angle [degrees]
+        solar_azimuth : numeric
+            Solar azimuth angle [degrees]
+
+        Returns
+        -------
+        orientation : dict-like
+            A dict-like object with keys `'surface_tilt', 'surface_azimuth'`
+        """
+        pass
+
+
+@dataclass
+class FixedMount(AbstractMount):
+    """
+    Racking at fixed (static) orientation.
+
+    Parameters
+    ----------
+    surface_tilt : float, default 0
+        Surface tilt angle. The tilt angle is defined as angle from horizontal
+        (e.g. surface facing up = 0, surface facing horizon = 90) [degrees]
+
+    surface_azimuth : float, default 180
+        Azimuth angle of the module surface. North=0, East=90, South=180,
+        West=270. [degrees]
+    """
+
+    surface_tilt: float = field(default=0)
+    surface_azimuth: float = field(default=180)
```

Comment:
why use `field` here? I thought this was equivalent to `surface_tilt: float = 0`. Might want to use `0.0` for type consistency.